docs/api-docs/storefront/widgets/widgets-tutorial.md

```diff
@@ -133,8 +128,7 @@ Accept: application/json
 <div class="HubBlock-content">
 
 > ### Note
-> * The response contains the `uuid` of the widget template. Make note of it; its used to create the widget later.
-> * Widget templates accept [Handlebars expressions](https://developer.bigcommerce.com/stencil-docs/reference-docs/handlebars-helpers-reference).
+> * Make note of the `uuid` of the widget template in the response. You will use it to create the widget in the next step.

```

Comment:
* Make a note of the `uuid` of the widget template in the response. You will use it to create the widget in the next step.